E2_ESRGAN/lib/utils.py

```diff
@@ -28,6 +29,40 @@ def load_checkpoint(checkpoint, training_phase, assert_consumed=True):
       status.assert_consumed()
 
 
+def interpolate_generator(generator_fn, discriminator, alpha):
+  r""" Refer to Section 3.4 of https://arxiv.org/pdf/1809.00219.pdf (Xintao et. al.)
```

Comment:
if I don't give r, the editor shows, the backslashes for the LaTeX maths, will be considered as escape sequences. any suggestions how to prevent that?

Reply:
Just add the paper name & authors (like they do in publications) in a "References" section.